backend/functions/lib/use-cases/case.assignment.service.ts

```diff
@@ -0,0 +1,48 @@
+import { ICaseAssignmentRepository } from '../interfaces/ICaseAssignmentRepository';
+import { getAssignmentRepository } from '../factory';
+import { CaseAttorneyAssignment } from '../adapters/types/case.attorney.assignment';
+import { ApplicationContext } from '../adapters/types/basic';
+import { TrialAttorneyAssignmentResponse } from '../adapters/types/trial.attorney.assignment.response';
+
+export class CaseAssignmentService {
+  private _assignmentRepository: ICaseAssignmentRepository;
+
+  constructor(assignmentRepository?: ICaseAssignmentRepository) {
+    if (!assignmentRepository) {
+      this._assignmentRepository = getAssignmentRepository();
+    } else {
+      this._assignmentRepository = assignmentRepository;
+    }
+  }
+
+  async createAssignment(
```

Comment:
Add functionality for the following in the business use case:
If an assignment has already been created on a case, and another assignment request for the same case comes in, respond with a 400 bad request.

Reply:
Done. Added functionality to check if an assignment already exists and throw an error in this scenario.